File: src/main/java/com/freya02/botcommands/api/commands/prefixed/annotations/Description.java
package com.freya02.botcommands.api.commands.prefixed.annotations

/**
 * Specifies the global help description for this command.
 */
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
annotation class Description(val value: String)

File: src/main/kotlin/io/github/freya022/botcommands/api/commands/prefixed/annotations/Description.kt
package io.github.freya022.botcommands.api.commands.prefixed.annotations

/**
 * Specifies the global help description for this command.
 */
@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.RUNTIME)
annotation class Description(val value: String)
